src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -242,10 +252,16 @@ public partial interface IWebProxy
     public partial class NetworkCredential : System.Net.ICredentials, System.Net.ICredentialsByHost
     {
         public NetworkCredential() { }
+        [System.CLSCompliantAttribute(false)]
+        public NetworkCredential(string userName, System.Security.SecureString password) { }
+        [System.CLSCompliantAttribute(false)]
+        public NetworkCredential(string userName, System.Security.SecureString password, string domain) { }
         public NetworkCredential(string userName, string password) { }
         public NetworkCredential(string userName, string password, string domain) { }
         public string Domain { get { throw null; } set { } }
         public string Password { get { throw null; } set { } }
+        [System.CLSCompliantAttribute(false)]
+        public System.Security.SecureString SecurePassword { get { throw null; } set { } }
```

Comment:
I don't think we should be adding the `SecureString` overloads to this class yet until the implementation PR for this is done.